images/postgres/config/main.tf

```diff
@@ -9,11 +9,26 @@ variable "extra_packages" {
   default     = ["postgresql-15", "postgresql-15-client", "postgresql-15-oci-entrypoint", "postgresql-15-contrib", "libpq-15"]
 }
 
+variable "uid" {
+  description = "The user ID of the postgres user."
+  default     = 70
+}
+
+variable "gid" {
+  description = "The group ID of the postgres group."
+  default     = 70
+}
+
+variable "run-as" {
```

Comment:
I think this always needs to run as 0?

Reply:
for bitnami type of version of it, it could be 1001 because afaik it didn't get running state unless you specify user rather than root?

Reply:
for now I'm converting it as draft since we missed the deadline of the postgres-bitnami image request, we can do that kind of refactoring later maybe.